hw/ip/prim/dv/prim_alert/data/prim_alert_testplan.hjson

```diff
@@ -0,0 +1,46 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "prim_alert"
+  testpoints: [
+    {
+      name: prim_alert_test
+      desc: '''Verify the prim_alert_sender and prim_alert_receiver pair.
+
+            This direct test includes five test sequences:
```

Comment:
These are discrete enough 'features' to be separated into 5 different testpoints. WDYT?